tests/unit/selectmenu/selectmenu_methods.js

```diff
@@ -78,21 +78,21 @@ asyncTest( "refresh - change selected option", function() {
 	var element = $( "#speed" ).selectmenu(),
 		button = element.selectmenu( "widget" );
 
-	equal( element.find( "option:selected" ).text(), button.text(), "button text after init" );
+	equal( " " + element.find( "option:selected" ).text(), button.text(), "button text after init" );
 
 	button.simulate( "focus" );
 	setTimeout(function() {
-		equal( element.find( "option:selected" ).text(), button.text(), "button text after focus" );
+		equal( " " + element.find( "option:selected" ).text(), button.text(), "button text after focus" );
 
 		element[ 0 ].selectedIndex = 0;
 		element.selectmenu( "refresh" );
-		equal( element.find( "option:selected" ).text(), button.text(),
+		equal( " " + element.find( "option:selected" ).text(), button.text(),
```

Comment:
I can't tell why both the actual and the expected argument are calculated. The expected argument should be a string literal. Same ordering issue as below. Probably should address that in a separate PR...